log.js

```diff
@@ -189,7 +189,17 @@ log.log = function (lvl, prefix, message) {
     }
   }
   if (stack) a.unshift(stack + '\n')
-  message = util.format.apply(util, a)
+
+  // Make sure the depth is limited to two without manipulating the default
+  // settings for the user.
+  if (util.inspect.defaultOptions) {
+    const tmpDepth = util.inspect.defaultOptions.depth
+    util.inspect.defaultOptions = { depth: 2 }
+    message = util.format.apply(util, a)
+    util.inspect.defaultOptions = { depth: tmpDepth }
+  } else {
+    message = util.format.apply(util, a)
+  }
 
```

Comment:
Invocations need to be wrapped in a try-finally or it could impact other packages if caught higher up since it's touching a globally exposed options object.